spec/inputs/s3_spec.rb

```diff
@@ -23,7 +23,7 @@
       "sincedb_path" => File.join(sincedb_path, ".sincedb")
     }
   }
-  let(:cutoff) { LogStash::Inputs::S3::CUTOFF_SECOND }
+  let(:cutoff) { 3 }
```

Comment:
Let's use a value in tests different from the default to make sure the value is respected by the plugin:

```suggestion
  let(:cutoff) { 4 }
```